changelog.d/10963.misc

```diff
@@ -0,0 +1 @@
+Fix the test utility function `create_room_as` so that `is_public=True` will explicitly set the `visibility` parameter of room creation requests to `true`. Contributed by @AndrewFerr.
```

Comment:
I think that `true` should be `public`?

Reply:
```suggestion
Fix the test utility function `create_room_as` so that `is_public=True` will explicitly set the `visibility` parameter of room creation requests to `public`. Contributed by @AndrewFerr.
```

Reply:
Oops, yes, thanks!